recipes/nand2tetris-core

```diff
@@ -0,0 +1,3 @@
+(nand2tetris-core :fetcher github
+                  :repo "CestDiego/nand2tetris.el"
+                  :files ("nand2tetris-core.el"))
```

Comment:
Would anybody use `nand2tetris-core` without `nand2tetris` or `nand2tetris-assembler`? This packaging feels too fine-grained to me, with *perhaps* the exception of `company-nand2tetris`.